src/cli.c

```diff
@@ -448,9 +450,27 @@ main(int argc, char** argv)
          else
          {
             /* Remote connection */
-            if (pgagroal_connect(host, atoi(port), &socket))
+
+            l_port = strtol(port, NULL, 10);
+            if ((errno == ERANGE
+                 && (l_port == LONG_MAX || l_port == LONG_MIN))
+                || (errno != 0 && l_port == 0))
```

Comment:
Could you put `&&` and `||` on the previous lines ?

Reply:
fixed